### ISSUE TITLE
[FEATURE] CCV Alumni Section

### ISSUE THREAD

**Issue by paulstey** [brown-ccv/ccv-website-hugo#305]
What would you all think about having an `Alumni` section on our website for our team members that have moved on? This could be our interns as well as folks from our regular team. 

What do you all think?

**Comment by fernandogelin** [comment]
sounds good, but should be introduced with website 2.0.

**Comment by mcmcgrath13** [comment]
> sounds good, but should be introduced with website 2.0.

Agreed.  We should also ask people as part of off-boarding (or retroactively) if they'd like to be in it
